### ISSUE TITLE
pCloud: EU data center not working

### ISSUE THREAD

**Issue by dmorlitz** [x0b/rcx#98]
Unable to create a pCloud backend when my account is in the EU data center.

I have recently moved my pCloud account from the USA to the EU. RCX had worked to access my account when I was using the USA data center. When I went to reauthenticate my account (since the data center change reset my OAuth ID) I am no longer able to create a pCloud backend.

I did try importing a working rclone.conf file for the new EU data center and that did not work either.

#### Technical Information

```
rclone: v1.51.0
rcloneExplorer: 1.11.4
android-sdk: 30
android-device: Pixel 3 XL
android-ndk-abis: arm64-v8a;armeabi-v7a;armeabi
```

**Comment by x0b** [comment]
[It looks like this has already been fixed in rclone](https://github.com/rclone/rclone/commit/d4b2709fb097bc2b8dde9acdedd779757d4252e8). This will be fixed in the next version.

**Comment by daviddem1971** [comment]
Same issue here. I confirm that this is fixed and working in the current rclone version.

Patiently waiting for the next RCX release with the fix :)

**Comment by x0b** [comment]
Version `v1.12.0` has released with rclone `1.55.1`. Please update and test again. If you still have this issue, please [open a new thread in the rclone forum](forum.rclone.org/) or a new issue in the rclone repo.
